src/test/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilderTest.java

```diff
@@ -0,0 +1,88 @@
+package com.salesforce.dynamodbv2.mt.mappers.sharedtable;
+
+import static com.amazonaws.services.dynamodbv2.model.KeyType.HASH;
+import static com.amazonaws.services.dynamodbv2.model.ScalarAttributeType.S;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.model.AttributeDefinition;
+import com.amazonaws.services.dynamodbv2.model.CreateTableRequest;
+import com.amazonaws.services.dynamodbv2.model.GlobalSecondaryIndex;
+import com.amazonaws.services.dynamodbv2.model.KeySchemaElement;
+import com.amazonaws.services.dynamodbv2.model.Projection;
+import com.amazonaws.services.dynamodbv2.model.ProjectionType;
+import com.amazonaws.services.dynamodbv2.model.ProvisionedThroughput;
+import com.salesforce.dynamodbv2.dynamodblocal.AmazonDynamoDbLocal;
+import com.salesforce.dynamodbv2.mt.context.MtAmazonDynamoDbContextProvider;
+import com.salesforce.dynamodbv2.mt.context.impl.MtAmazonDynamoDbContextProviderThreadLocalImpl;
+import com.salesforce.dynamodbv2.mt.mappers.sharedtable.impl.MtAmazonDynamoDbBySharedTable;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class SharedTableBuilderTest {
```

Comment:
I'm assuming you're not testing provisionedThroughput=0 because this test won't work against DynamoDBLocal?

Reply:
Yes, the tests for provisionedThroughput=0 are in /src/integration/java/../SharedTableBuilderIt, so they can be tested against a remote DynamoDB. DynamoDBLocal chokes when provisionedThroughput is not set; provisioned throughput cannot be set if billing mode is set to PAY_PER_REQUEST, thus we can only test it against remote.